scheds/rust/scx_layered/src/bpf/cost.bpf.c

```diff
@@ -363,7 +363,7 @@ __weak int has_budget(struct cost *costc, struct layer *layer)
 /*
  * Initializes all budgets.
  */
-static void initialize_budgets(u64 refresh_intvl_ns)
+int initialize_budgets(u64 refresh_intvl_ns)
```

Comment:
Do we have a negative view of global functions? I guess being explicit about it either way isn't ideal because it limits what the compiler can do, but for init functions global seems fine forever.